### ISSUE TITLE
P2P program bug on laptops

### ISSUE THREAD

**Issue by Dropshotz** [Yawn-Wider/YWPolarisVore#209]
#### Brief description of the issue
Deleting P2P program whilst having a server up will forever leave the file shared up

#### What you expected to happen
The file to disappear

#### What actually happened
Can still download it!

#### Steps to reproduce
- Get a file and P2P to share it on a laptop/tablet
- Share file
- Delete p2p as server is up
- File is still there to download

#### Code Revision
- (Found using the "Show Server Revision" verb under the OOC tab.)
 Server revision:  - 2019-10-11
6fed8381ea28bb17faa3f1758367739c879dc834 

#### Anything else you may wish to add:
- Potentially very harmful if done with certain software as it will become public use.

**Comment by DeepIndigo** [comment]
A lot of software is ID-locked isn't it? Like the ID mod software.

**Comment by Hawk-v3** [comment]
Yes, but the computers are ID locked as well, it's a bit strange.

Still, this just needs to call the server to close as it's being deleted, I guess.
